servers.json

```diff
@@ -1421,7 +1421,11 @@
             "minecraftparty.com",
             "mcctf.com",
             "mcpvp.com",
-            "minecraftpvp.com"
+            "minecraftpvp.com",
+            "mcctf.com",
```

Comment:
I'm confused, why'd you add mcctf.com & mcpvp.com again?

Reply:
Just incompetence on my end, fixed! Apologies

Reply:
> Just incompetence on my end, fixed! Apologies ([d17b043](https://github.com/LunarClient/ServerMappings/commit/d17b0435d7814bbfca4897f2c23d864f2a70ba51))

you're all good!! Thanks!